s3sync.go

```diff
@@ -144,7 +149,7 @@ func (m *Manager) syncLocalToS3(chJob chan func(), sourcePath string, destPath *
 	wg := &sync.WaitGroup{}
 	errs := &multiErr{}
 	for source := range filterFilesForSync(
-		listLocalFiles(sourcePath), m.listS3Files(destPath), m.del,
+		listLocalFiles(sourcePath), m.listS3Files(destPath), m.del, m.softDelete,
```

Comment:
Should we check somewhere that both del and softDelete are not defined.

Reply:
For sure, I was thinking of even disabling delete in case softdelete is defined and printing out a warning, because I would expect a user who sets soft delete to not want to accidentally hard delete stuff, but a user who wants to hard delete stuff doesn't care if there's a mistake of once tagging things for soft delete and having to run the thing again.

Reply:
Hmm, I guess that wouldn't work as the order of the options can be whatever. I'll just check that both are not set :).